src/main/scala/sigmastate/serialization/OpCodes.scala

```diff
@@ -191,5 +186,11 @@ object OpCodes extends ValueCodes {
   val CollRotateRightCode     : OpCode = (LastConstantCode + 141).toByte
 
   val ContextCode             : OpCode = (LastConstantCode + 142).toByte
-  val XorOfCode               : OpCode = (LastConstantCode + 143).toByte
+  val XorOfCode               : OpCode = (LastConstantCode + 143).toByte // equals to 255
+
+  
+  /** This is the maximum valid opcode in the current version.
+    * It is used by soft-forkability checks.
+    */
+  val MaxOpCode             : OpCode = XorOfCode
```

Comment:
For now it's usage is unclear, as soon as we have gaps between opcodes and soft-forkability is not implemented yet

Reply:
The intention was to compress opcodes, but since we don't do this, this can be removed.

Reply:
remove please

Reply:
removed